### DISCUSSION TITLE
backup email address and backup phone

### DISCUSSION THREAD

**Discussion by ShovanSaha2020** [simple-login/app#818] [Feature Requests]
If there were options to add a backup email address or a backup phone number, it would have been amazing.

**Comment by 0xspade** [comment]
this feature is debate-able bro since SimpleLogin should not ask for additional PII information to identify you just in case "if-all-else-fail", you know what I mean. What matters is your privacy :)

**Reply by c0nfigurati0n** [reply]
lol they already have all of you email xD so this statement doesn't make any senes.

**Comment by ShovanSaha2020** [comment]
They already have my IP address, device information and email address which
is enough to identify me.

**Reply by c0nfigurati0n** [reply]
More or less, yeah. Altho i'm not sure about the device information part.
